app/assets/javascripts/discourse/app/controllers/user-private-messages.js

```diff
@@ -23,6 +24,51 @@ export default class extends Controller {
   @readOnly("router.currentRoute.parent.name") currentParentRouteName;
   @readOnly("site.can_tag_pms") pmTaggingEnabled;
 
+  get messagesDropdownValue() {
+    const parentRoute = this.router.currentRoute.parent;
+
+    if (Object.keys(parentRoute.params).length > 0) {
```

Comment:
Just checking, can `parentRoute` ever be `null` here?

Reply:
Not possible based on the current routing structure when this controller is rendered :)